cdap-docs/developer-manual/source/pipelines/developing-plugins/presentation-plugins.rst

```diff
@@ -687,6 +687,34 @@ CDAP pipelines as of version |version|.
             }
           }
 
+   * - ``toggle``
+     - - ``on``: 'On' state of the toggle, defined by ``value``: value to be set for the plugin property, and
+         ``label``: a label to be rendered in UI for the 'On' state, limited to 64 characters.
```

Comment:
I think ``for the 'On' state`` is redundant, since you already start the sentence with ``'On' state of the toggle, defined by...``